packages/cli/src/commands/export/credentials.ts

```diff
@@ -105,13 +105,12 @@ export class ExportCredentialsCommand extends BaseCommand {
 			}
 		}
 
-		const findQuery: IDataObject = {};
+		const findQuery: FindOptionsWhere<ICredentialsDb> = {};
 		if (flags.id) {
 			findQuery.id = flags.id;
 		}
 
-		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		const credentials = await Db.collections.Credentials.find(findQuery);
+		const credentials = await Db.collections.Credentials.findBy(findQuery);
```

Comment:
this line was the culprit. in typeorm 0.2.x `.find` worked differently than it does in `0.3.x`.